module/VuFind/src/VuFind/Search/Solr/Params.php

```diff
@@ -489,6 +489,17 @@ protected function normalizeSort($sort)
                 );
             }
         }
+        $tieBreaker = $this->getOptions()->getSortTieBreaker();
```

Comment:
I had originally thought about allowing the tie-breaker to be a fully-developed sort value -- e.g. comma-separated, potentially including asc/desc, potentially using the legacy aliases. However, on seeing this elegant solution to the problem, I'm somewhat convinced that we don't need to do all that extra work. We could always start with just a field name for now, and we could expand the functionality in the future if a use case was identified to justify the work (since a future fuller interpretation of sort fields would still be backward compatible with configuration consisting of just the name of a single Solr field).

What do you think? Is it worth adding any additional functionality here, or should this stand as-is? We can ask for more opinions once you're ready to take this out of draft mode.